server/api/src/rsGeneralAdmin.cpp

```diff
@@ -316,12 +325,12 @@ int _remove_rebalance_timestamp_avu_from_resource(
             free( modAVUMetadataInp.arg5 );
         }};
         // remove AVU from resource
-        modAVUMetadataInp.arg0 = strdup( "rmw" );
+        modAVUMetadataInp.arg0 = strdup("rm");
         modAVUMetadataInp.arg1 = strdup( "-R" );
         modAVUMetadataInp.arg2 = strdup( _resource_name.c_str() );
         modAVUMetadataInp.arg3 = strdup( "rebalance_operation" );
-        modAVUMetadataInp.arg4 = strdup( "%" );
-        modAVUMetadataInp.arg5 = strdup( "%" );
```

Comment:
The original removed many AVUs. Does the new code still cover that?

Another question is - should it remove multiple AVUs?

Reply:
I actually think this is more correct than removing all of the rebalance_operation AVUs. If multiple rebalance_operation AVUs appear on a resource, that means that multiple rebalance operations were started and each annotated its own metadata, which is a bug. The whole point of the AVU is to prevent multiple rebalances from occurring, so if that ever happens, this at least makes the bug more visible. Thoughts?

Reply:
@trel may have more insight into this. Here's the original issue from which this feature came: https://github.com/irods/irods/issues/3683

I don't see any specific requirements around removing all of the rebalance_operation AVUs once a rebalance completes. Again, the fact that multiple rebalances can run simultaneously is itself a bug.

Reply:
Correct - the AVU itself is the 'lock' for a rebalance running.  So there should / cannot be more than one.

Reply:
All of that sounds great.

Resolving.